src/pages/Tally/TallyLayout.tsx

```diff
@@ -162,13 +169,8 @@ const TallyLayout = () => {
         throw Error('registeredBallotsFileName is undefined')
       }
 
-      // if we have no ballots, explicitly
-      // bypass running the tally
-      // TODO: Execute API with zero count
```

Comment:
we should keep the TODO here since we should technically actually call into the API to verify the zero result

Reply:
See this:
https://github.com/microsoft/electionguard-admin-device/issues/63